pom.xml

```diff
@@ -161,8 +161,8 @@
                 <version>2.3.2</version>
                 <configuration>
                     <encoding>UTF-8</encoding>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.7</source>
+                    <target>1.7</target>
```

Comment:
This change to the compiler target isn't really needed for the plugin itself, but the code generated does require 1.7, since String based switch statements are being employed.